### ISSUE TITLE
Copied pastures do not appear in correct order

### ISSUE THREAD

**Issue by LisaMoore1** [bcgov/range-web#620]
These were entered in the following order:  Original, Manual Pasture(1), then copy of original

![image.png](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/e208cc08-119b-456b-ba64-4eba0c05c9d9)

**Comment by calebissharp** [comment]
@LisaMoore1 Is this reordering on save?

**Comment by LisaMoore1** [comment]
@calebissharp I’m not 100% sure.  I just noticed it while I was doing a bunch of pasture copying and deleting.  I think I noticed it right after I copying a pasture but before doing anything else.  Whether that is when it occurred or not I’m not sure.  Sorry.

**Comment by calebissharp** [comment]
@LisaMoore1 Does the resorting just happen when you **copy** a pasture?

**Comment by LisaMoore1** [comment]
@calebissharp Looks like only after a save has occurred after copying a pasture. 
- manual pasture added -->good order
- save --> still good order
- copy pasture --> correct order
- save --> copied pasture moves to somewhere up the list

**Comment by calebissharp** [comment]
@LisaMoore1 Does this happen for other fields that allow you to copy?

**Comment by LisaMoore1** [comment]
@calebissharp I think this is the only place we have a straight up copy function so the only place I can test it.

**Comment by calebissharp** [comment]
@LisaMoore1 What about grazing schedules/rows, additional requirements, and monitoring areas?

**Comment by LisaMoore1** [comment]
@calebissharp Thanks for having me look at this. Got foggy brain. I thought we could copy in all sorts of places but when I looked today this is where we're at:

Additional Requirements -- has the same order problem for copied entries as pastures (missed this one earlier)
Monitoring Areas -- see Issue #627
Schedules are copied to a specific year and sorted by year so not an issue there. 
Schedule rows seem to be staying at order entered even when duplicating rows
Minister's Issues -- don't have copy function
Management Considerations -- don't have copy function

**Comment by calebissharp** [comment]
@LisaMoore1 Ok great. Thanks for double checking. I think the monitoring areas should be fixed by #626, and the pastures by #624. I'll have to open another PR for additional requirements though.

**Comment by calebissharp** [comment]
Oh nevermind, additional requirements should have also been fixed.

**Comment by LisaMoore1** [comment]
@calebissharp This is still happening for me. Has the code been merged?

**Comment by calebissharp** [comment]
@LisaMoore1 Yep.

**Comment by calebissharp** [comment]
@LisaMoore1 Is is possible you're using an outdated version of the app. Can you try clearing cache?
